draft-ietf-quic-multipath.md

```diff
@@ -554,39 +551,21 @@ ACK frames are received in 1-RTT packets while the state of multipath
 negotiation is ambiguous, they MUST be interpreted as acknowledging
 packets sent on path 0.
 
-Endpoints negotiate the use of one packet number space for all paths or
-separate packet number spaces per path during the connection handshake
-{{nego}}. While separate packet number spaces allow for more efficient
-ACK encoding, especially when paths have highly different latencies,
-this approach requires the use of a connection ID. Therefore use of
-a single number space can be beneficial when endpoints use zero-length
-connection ID for less overhead.
-
-## Using One Packet Number Space
-
-If the multipath option is negotiated to use one packet number space
-for all paths, the packet sequence numbers are allocated from the common
-number space, so that, for example, packet number N could be sent
-on one path and packet number N+1 on another.
-
-ACK frames report the numbers of packets that have been received so far,
-regardless of the path on which they have been received. That means
-the senders needs to maintain an association between sent packet numbers
-and the path over which these packets were sent. This is necessary
-to implement per path congestion control.
-
-When a packet is acknowledged, the state of the congestion control
-MUST be updated for the path where the acknowledged packet
-was originally sent. The RTT is calculated based on the delay
-between the transmission of that packet and its first acknowledgement
-(see {{compute-rtt}}) and is used to update the RTT statistics
-for the sending path.
-
-Also loss detection MUST be adapted to allow for different RTTs on
-different paths.  For example, timer computations should take into
-account the RTT of the path on which a packet was sent.  Detections
-based on packet numbers shall compare a given packet number to the
-highest packet number received for that path.
+## Using Zero-Length connection ID {#using-zero-length}
+
+If a node elects to use zero-length connection IDs, it MUST implement the
+handling of Acknowledgements defined in {{sending-acknowledgements-and-handling-ranges}}.
+The peer node using zero-length connection IDs SHOULD implement a
+logic to handle loss detection in the presence of multiple paths,
+identify the path on which packets are acknowledged or determined
+lost for the purpose of congestion control as explained
+in {{zero-length-cid-loss-and-congestion}}, mitigate the ECN
+issues mentioned in {{ecn-and-zero-length-cid-considerations}}, and
+mitigate the RTT measurement issues explained
+in {{ack-delay-and-zero-length-cid-considerations}}. If a node
+does not support this logic, it MUST limit its use of multipath
+as explained in {{restricted-sending-to-zero-length-cid-peer}}.
```

Comment:
I don't think the MUST and SHOULD here make a lot of sense, given there are more/other normative language later in this section. Also ice think your removed too much.... here is an alternative proposal for this section intro part:
```suggestion
If a zero-length connection ID is used, one packet number space
 for all paths. That means the packet sequence numbers are allocated from the common
 number space, so that, for example, packet number N could be sent
 on one path and packet number N+1 on another.
 
 In this case, ACK frames report the numbers of packets that have been received so far,
 regardless of the path on which they have been received. That means
 the senders needs to maintain an association between sent packet numbers
 and the path over which these packets were sent. This is necessary
 to implement per path congestion control, as explained
in {{zero-length-cid-loss-and-congestion}}.

Further, the receiver of packets with zero-length connection IDs should
handling of acknowledgements defined in {{sending-acknowledgements-and-handling-ranges}}.

ECN handing is discussed in {{ecn-handling}}, and
mitigation of the RTT measurement is further explained
in {{ack-delay-and-zero-length-cid-considerations}}. 

If a node
does not support this logic, it MUST limit its use of multipath
as explained in {{restricted-sending-to-zero-length-cid-peer}}.
```